### PR TITLE
feat: uffizzi integration

### PR DESCRIPTION
**Pull request by daramayis**
## Uffizzi integration

### CODE REVIEW

File: uffizzi/DockerfileGo [hunk 1]
```diff
@@ -0,0 +1,14 @@
+FROM golang:1.18 as build 
+
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get -y install openssh-server sudo
```
Comment (axisofentropy) [inline uffizzi/DockerfileGo]: Please remove the `apt` cache as seen here https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run

File: .github/workflows/uffizzi-preview.yml [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    name: Use Remote Workflow to Preview on Uffizzi
+    needs:
+      - cache-compose-file
+    uses: UffizziCloud/preview-action/.github/workflows/reusable.yaml@v2.6.1
+    with:
+      # If this workflow was triggered by a PR close event, cache-key will be an empty string
+      # and this reusable workflow will delete the preview deployment.
+      compose-file-cache-key: ${{ needs.cache-compose-file.outputs.compose-file-cache-key }}
+      compose-file-cache-path: docker-compose.rendered.yml
+      server: https://app.uffizzi.com/
```
Comment (axisofentropy) [inline .github/workflows/uffizzi-preview.yml]: Remove trailing `/` from URL.

File: uffizzi/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+COPY go.mod go.sum ./
+RUN go mod download
+COPY . .
+RUN CGO_ENABLED=0 GOOS=linux go build
+
+FROM alpine:3.15
+RUN apk add --no-cache -U git xdg-utils
+WORKDIR /go/src/github.com/jesseduffield/lazygit/
+COPY --from=build /go/src/github.com/jesseduffield/lazygit ./
+COPY --from=build /go/src/github.com/jesseduffield/lazygit/lazygit /bin/
```
Comment (axisofentropy) [inline uffizzi/Dockerfile]: This section looks redundant and doesn't get used?

File: .github/workflows/uffizzi-preview.yml (hunk 2 above)
Comment (daramayis) [reply .github/workflows/uffizzi-preview.yml]: Done.

File: uffizzi/Dockerfile (hunk 3 above)
Comment (daramayis) [reply uffizzi/Dockerfile:14]: Fixed, with new docker image

Comment (daramayis) [reply uffizzi/Dockerfile]: Right, fixed.

File: uffizzi/DockerfileGo (hunk 1 above)
Comment (daramayis) [reply uffizzi/DockerfileGo]: Done

**Pull request closed without merging**